Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
Because I'm a Sonata member, which grants me write access to all Sonata projects. It does not mean that I have to learn Propel before reviewing this though. Maybe another Sonata member can vouch for this kind of modification, but in the meantime, I thought you would be glad to get a thorough review of your changes. Was I wrong to think that?